packages/trackerless-network/src/logic/StreamrNode.ts

```diff
@@ -216,10 +220,10 @@ export class StreamrNode extends EventEmitter<Events> {
             forwardingPeer
         )
         entryPoints = knownEntryPoints.concat(discoveryResult.discoveredEntryPoints)
-        await layer1.joinDht(sampleSize(entryPoints, 4))
+        await layer1.joinDht(sampleSize(entryPoints, NETWORK_SPLIT_AVOIDANCE_LIMIT), false, enableRejoins)
         await this.streamEntryPointDiscovery!.storeSelfAsEntryPointIfNecessary(
             streamPartId,
-            discoveryResult.joiningEmptyStream,
+            layer1.getBucketSize() < NETWORK_SPLIT_AVOIDANCE_LIMIT,
```

Comment:
Is `discoveryResult.joiningEmptyStream` used elsewhere? If not it could be removed along with the logic to calculate it.